src/main/java/uk/nhs/hee/tis/revalidation/connection/repository/CurrentConnectionElasticSearchRepository.java

```diff
@@ -32,7 +32,7 @@
 public interface CurrentConnectionElasticSearchRepository
     extends ElasticsearchRepository<CurrentConnectionsView, String> {
 
-  @Query("{\"bool\":{\"must_not\":{\"match\":{\"membershipType\":\"MILITARY\"}},"
+  @Query("{\"bool\":{\"must_not\":{\"match\":{\"placementGrade\":\"F1\"}},"
```

Comment:
I think we need to filter out both military membershipType and F1 placementGrade.

Reply:
Ooh good catch, I think the minifier I used must have removed the "duplicate" "must_not"